deployment/update_production.sh

```diff
@@ -31,6 +31,9 @@ set -x # print executed commands. enable this here to not print the if above.
 
 sudo -H -u $USERNAME git fetch
 
+# install sass
+./deployment/install_dart_sass.sh
+
```

Comment:
I think this is not the right place, at least we don't have e.g. a pip install in here either. I suppose a "setup production" script simply doesn't exist at the moment.

Reply:
Hm, so should this only be documented somewhere else (e.g. in the wiki) or should there be a new `setup_production.sh`?

Reply:
the number of times we had to spin up a new production instance in the last 8 years has been zero i think :D if we ever need to do that, the vagrant provision script should contain mostly complete instructions.

speaking of which, could you add this command there? :)

Reply:
I now inlined the content of `install_dart_sass.sh` into `provision_vagrant.sh`. Just to be sure – the command for installing dart has always have been in `provision_vagrant.sh`, so I don’t know whether I understood your last question correctly?

Reply:
sorry, i thought you added it only to update_production.sh. I think inlining into provision_vagrant.sh is fine until we need it in another place.

Reply:
seems like you need it for the new travis.yml in the typescript PR...?

Reply:
Good catch, then I’ll revert to the version with an extra shell script